notebooks/Image_Analysis.ipynb

```diff
@@ -83,7 +83,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "bg_vars, fg_vars = [], []"
+    "bg_vars, fg_vars = [], []\n",
+    "#Check for NANs or 0s, if present remove them"
    ]
   },
   {
```

Comment:
On line 96 `img_id in tqdm(img_ids[:10000]):` by default should cycle through all the `img_id` not just the first 10000.

Reply:
Done.